x-pack/test_serverless/functional/page_objects/svl_search_connectors_page.ts

```diff
@@ -40,6 +40,12 @@ export function SvlSearchConnectorsPageProvider({ getService }: FtrProviderConte
         await testSubjects.existOrFail('serverlessSearchCancelNameButton');
         await testSubjects.setValue('serverlessSearchEditNameFieldText', name);
         await testSubjects.click('serverlessSearchSaveNameButton');
+        await retry.waitForWithTimeout('edit name form to disappear', 2000, () =>
```

Comment:
Ideally we do this without a specific time timeout because that's annoying. Maybe @TattdCodeMonkey has a better idea?

Reply:
@sphilipse The missingOrFail has a [default timeout of 2.5 seconds](https://github.com/elastic/kibana/blob/5d9b7e7554d98f713efd0e82d89513a492a154a0/packages/kbn-ftr-common-functional-ui-services/services/test_subjects.ts#L84).  I thought of adding the extra 2 second timeout,  hoping that this would give some extra time to wait until form disappears.  

With only missingOrFail (without 2 second timeout) the FTR **doesn't** fail,  tested in Prod & QA deployments.

Reply:
Okay, let's see if this fixes it :)

Reply:
cool. I will merge this PR as it is then. Thanks 🙌